app/services/feature_flag.rb

```diff
@@ -28,6 +28,7 @@ def feature
     admin_change_materials
     user_research_full_booked
     sit_user_research_fully_booked
+    actual_dqt_api
```

Comment:
😆

Reply:
Name suggestions are also welcome before we set it in stone

Reply:
`full`?

Reply:
How do the people writing those apis distinguish the two? Maybe we can borrow naming from them?

Reply:
I don't see much from outside. the only maybe useful bit are from the hostname which contains `dqt`, `crm4`, `dynamics`

Reply:
I've just mass renamed everything to `FullDqt` and `full_dqt_api`